packages/theme-wizard-app/src/components/wizard-colorscale-input/index.ts

```diff
@@ -190,16 +197,14 @@ export class WizardColorscaleInput extends WizardTokenInput {
             id=${this.#idColor}
             name=${this.#idColor}
             type="color"
-            value=${this.colorToken
-              ? legacyToModernColor.encode(resolveColorValue(this.colorToken) ?? parseColor('black'))
-              : ''}
+            .value=${this.currentColorValue}
             colorSpace=${this.colorSpace}
             @change=${this.handleColorChange}
             list="preset-colors"
           />
         </div>
-        <output
```

Comment:
in case you're wondering: the `output` element implicitly has a `role=status` and this both interferes with other status elements (like for scraper form) and seems unhelpful to be rendered 14 times in a row with hardly useful content.